src/strings/csv/en.csv

```diff
@@ -622,7 +622,7 @@ NURSERY_REQUIRED,Nursery *
 NURSERY_TRANSFER,Nursery Transfer
 OBSERVATION_IN_PROGRESS,Observation in Progress
 OBSERVATION_STATUS,Observation Status
-OBSERVATION_STATUS_OUTSTANDING,Outstanding,In the sense of Pending or Unresolved
+OBSERVATION_STATUS_OUTSTANDING,Outstanding,Remaining to be done
```

Comment:
Nope did not work. Getting this in for the other strings.
Something we need to follow up on @charlie-tf 
Maybe this can be something other than 'Outstanding' .. maybe 'Pending'?